enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/StandardEnterpriseAuthSubject.java

```diff
@@ -71,7 +75,17 @@ public AuthenticationResult getAuthenticationResult()
     public void setPassword( String password, boolean requirePasswordChange )
             throws IOException, InvalidArgumentsException
     {
-        getUserManager().setUserPassword( (String) shiroSubject.getPrincipal(), password, requirePasswordChange );
+        try
+        {
+            getUserManager().setUserPassword( (String) shiroSubject.getPrincipal(), password, requirePasswordChange );
+            securityLog.info( this, "changed password%s",
+                    requirePasswordChange ? ", with password change required" : "" );
```

Comment:
I still believe in having the logging inside the user manager will simplify our lives..